test/config/config.go

```diff
@@ -32,4 +33,7 @@ func (c *CiliumTestConfigType) ParseFlags() {
 		"Provision Vagrant boxes and Cilium before running test")
 	flag.BoolVar(&c.HoldEnvironment, "cilium.holdEnvironment", false,
 		"On failure, hold the environment in its current state")
+	flag.BoolVar(&c.Developer, "cilium.developer", false,
```

Comment:
I'm not a fan of this flag name. "Developer" is not evocative of what it does. Can you clarify the documentation of it as well? Does it mean that if this flag is set, that cilium.holdEnvironment's variable will also be overridden? What if the user sets `cilium.holdEnvironment` to be false? This is not user-friendly. 
Please define what 'configured' means in this context as well. By 'configured' do you mean 'booted', i.e. `vagrant up <vm name>`?